cmd/server/assets/codestatus/index.html

```diff
@@ -0,0 +1,53 @@
+{{define "codestatus/index"}}
+
+{{$authApp := .authApp}}
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  {{template "head" .}}
+</head>
+
+<body>
+  {{template "navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>Check verification code status</h1>
+    <p>
+      Use a unique identifier to check the status of a verification code previously shared with your patient.
+    </p>
+
+    <div class="card mb-3">
+      <div class="card-body">
+        <form method="GET" action="codestatus/show">
```

Comment:
form is missing the CSRF protection key.
and that might be a reason to shift this to a POST

Reply:
Yea, this should be POST and use the CSRF field imo

Reply:
Done